### ISSUE TITLE
Control key support

### ISSUE THREAD

**Issue by double-fault** [Bone-Project/BoneOS#159]
If you press the <kbd>Control</kbd> key with another key, it triggers a shorcut. Here is a list of shortcuts I plan to support:

- [x] <kbd>Control</kbd> + <kbd>A</kbd> - go to the beginning of the line you are currently typing on.
- [x] <kbd>Control</kbd> + <kbd>E</kbd> - go to the end of the line you are typing on.
- [x] <kbd>Control</kbd> + <kbd>L</kbd> - clears the screen.
- [x] <kbd>Control</kbd> + <kbd>U</kbd> - clears the line you are typing in.
- [x] <kbd>Control</kbd> + <kbd>H</kbd> - backspace.
- [ ] <kbd>Control</kbd> + <kbd>C</kbd> - stops whatever you are running.
- [x] <kbd>Control</kbd> + <kbd>D</kbd> - exits the shell instance.
- [x] <kbd>Control</kbd> + <kbd>N</kbd> - opens a new shell instance.
- [x] <kbd>Control</kbd> + <kbd>W</kbd> - deletes the word before the cursor.
- [x] <kbd>Control</kbd> + <kbd>T</kbd> - swaps the last two characters before the cursor.

**Comment by srifqi** [comment]
Why don't use shortcut many people already know? (i.e. <kbd>Control</kbd> + <kbd>W</kbd> to close window (shell instance))

Also, can we make use of <kbd>Home</kbd> and <kbd>End</kbd> key?

**Comment by double-fault** [comment]
@srifqi to stop a shell instance do <kbd>Control</kbd>+<kbd>D</kbd>, that is the *nix standard. What do you want to use the "Home" and "End" key for?

**Comment by srifqi** [comment]
<kbd>Home</kbd> - go to the beginning of the line you are currently typing on.
<kbd>End</kbd> - go to the end of the line you are typing on.

**Comment by double-fault** [comment]
@srifqi okay, can make those aliases of <kbd>Control</kbd>+<kbd>A</kbd> and <kbd>Control</kbd>+<kbd>E</kbd> respectively.

**Comment by amanuel2** [comment]
Plan on finishing?

**Comment by double-fault** [comment]
@amanuel2 nope, I'm extremely busy nowadays and my first priority is completing a refactor on FireAlarm.
